### ISSUE TITLE
Can't inject ILogger<T>

### ISSUE THREAD

**Issue by MartinRothschink** [thomasclaudiushuber/mvvmgen#32]
Hi Thomas,

great package, I'm using it already in two projects.

I like to inject an ILogger<t> in my view model, but that creates code that does not compile.

E.g.

`[Inject(typeof(ILogger<SettingsPageViewModel>))]`

Is that  a known issue? Or should that work?

Regards
Martin

**Comment by thomasclaudiushuber** [comment]
Hi @MartinRothschink ,

that should definitely work, but looks like injecting generic interfaces does not work yet.

I'll take a look at it, thank you for reporting it. (Issue #22 might refer to the same problem)

**Comment by thomasclaudiushuber** [comment]
Hi @MartinRothschink ,

OK, I see, the generated property name contains in this case a closing angle bracket >, and so the code does not compile.

I'll create a fix for this. For now, a workaround would be to explicitly specify the property name with the Inject attribute:

```
[Inject(typeof(ILogger<SettingsPageViewModel>),PropertyName ="Logger")]
```

**Comment by MartinRothschink** [comment]
Thanks Thomas,

will try the workaround.

**Comment by thomasclaudiushuber** [comment]
Fixed in release 1.1.4 that is now available via NuGet. Thanks for reporting this @MartinRothschink.

**Comment by Daimonion1980** [comment]
Perfect. 

Exact the same situation as i ran into this morning. 

Update nuget package from 1.1.3 to 1.1.4 and everything is fine again!!!! What a great work!
